_pages/en_US/troubleshooting.txt

```diff
@@ -155,3 +155,15 @@ Browser based exploits (such as browserhax or 2xrsa) are often unstable and cras
 1. Your device is bricked
 1. You will need to get an ntrboot-comptible flashcart (one of the ones on [this list](ntrboot) or a [hardmod](https://gbatemp.net/threads/414498/), or repair / replace your device
 1. For support (in English), join [Nintendo Homebrew on Discord](https://discord.gg/MWxPgEp)
+
+## WiFi connection errors after installing CFW
+
+1. You get an error while trying to use WiFi (during system update, loading the browser/Eshop, ect.)
+1. Try moving closer to your WiFi router, and rebooting your 3DS
+1. Try setting your DNS settings to "auto"
+1. Try rebooting your WiFi router
+1. Try deleting your WiFi connection/s, then sign into your Wifi again
+1. Try conecting to a different WiFi conection (mobile hotspot, ect.)
+1. Try again later (Nintendo servers are down, ect.)
+1. If this still gives you an error, [follow CTRTransfer](ctrtransfer), then try again
+1. For support (in English), join [Nintendo Homebrew on Discord](https://discord.gg/MWxPgEp)
```

Comment:
I don't think `Try` should be at the beginning of every sentence (use the form used with CTRTransfer, which is `then try again`). It's left unspoken what to do after you reboot you WiFi router/set DNS to auto/whatever, which is unclear.